metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1851,7 +1849,6 @@ class MetalsLanguageServer(
       check()
       buildTools
         .loadSupported()
-        .foreach(_.onBuildTargets(workspace, buildTargets))
```

Comment:
I'm confused by this, what was the `onBuildTargets` method here supposed to be doing?

Reply:
It's another leftover from Pants, came up when I added the explicits flag.